examples/entangled_states.ipynb

```diff
@@ -4,21 +4,28 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Entangled State Example\n",
+    "# Entangled States\n",
     "\n",
-    "This example shows how to enact and measure a graph state on a set of connected qubits that form a graph with the qubits as nodes and pairs of coupled qubits as edges. "
+    "In this notebook we explore the subset of methods in `entangled_states.py` that are related specifically to [graph states](https://en.wikipedia.org/wiki/Graph_state). Although it is worth noting that the module also alows one to create [GHZ](https://en.wikipedia.org/wiki/Greenberger%E2%80%93Horne%E2%80%93Zeilinger_state) states.\n",
+    "\n",
+    "\n",
+    "Graph states are a very simple entangled state construct as there are only two steps:\n",
+    "1. prepare all qubits on the desired lattice in the $|+\\rangle$ state \n",
```

Comment:
The \rangle is showing up quite large.

Reply:
That is weird. It looks fine in my notebook. hmmm ill let it slide now.